client/landing/gutenboarding/components/plans/plans-table/style.scss

```
@@ -1,84 +1,35 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../../mixins.scss';
 @import '../../../variables.scss';
+@import '../variables.scss';
 @import 'assets/stylesheets/shared/mixins/placeholder'; // Contains the placeholder mixin
 
-.plans-grid__table {
-	overflow-x: auto;
-	margin-bottom: 70px;
-
-	@include break-large {
-		overflow-x: initial;
-	}
-}
-
-// Make panning edge-to-edge
-.plans-grid__table {
-	width: calc( 100% + 30px + 30px );
-	margin-left: -30px;
-	margin-right: -30px;
-
-	@include break-mobile {
-		width: auto;
-		margin-left: 0;
-		margin-right: 0;
-
-		.plans-table {
-			padding: 0;
-		}
-	}
-}
-
 .plans-table {
 	display: flex;
 	flex-direction: row;
-	margin: 0 -4px;
-	@include break-xlarge {
-		margin: 0 -10px;
-	}
-
-	.plan-item {
-		flex-basis: 193px;
-		flex-grow: 0.5;
-		flex-shrink: 0;
-		margin: 0 4px;
-		margin-bottom: 24px;
-
-		&:first-child {
-			border-left: 30px solid transparent;
-		}
-
-		&:last-child {
-			border-right: 30px solid transparent;
-		}
-
-		@include break-large {
-			flex-shrink: 0.2;
+	width: 100%;
```

Check this out:

```suggestion
	width: 100%;
	scroll-snap-type: x mandatory;
```

This will make scrolling snap to children as if you have Swiper 🎉

---

Done.